webapp/src/app/core/services/block.service.ts

```diff
@@ -16,4 +18,20 @@ export class BlockService {
     delete this.blockMap[block.id];
     this.refreshChanges.next();
   }
+
+  addBlock(blockType: BlockType): void {
+    const newBlock = new BlockData(randstr64(10), blockType);
+    newBlock.order = this.blockNumber++;
+    newBlock.variableId = randCustomString(numerals, 4);
+    this.blockMap[newBlock.id] = newBlock;
+  }
+
+  getBlockList(): BlockData[] {
+    const blockList: BlockData[] = Object.values(this.blockMap);
+    blockList.sort((a, b) => Math.sign(a.order - b.order)); // Oldest first
+    // Refresh order
```

Comment:
`Refresh order` -> `Sorted order`?

Reply:
Good point. So maybe we should name it `maxBlockOrderValue` or something like that.

Reply:
Even better :)